lib/bandit/http1/adapter.ex

```diff
@@ -370,6 +385,8 @@ defmodule Bandit.HTTP1.Adapter do
           {body, headers, %{}}
       end
 
+    compress = Keyword.get(req.opts.http_1, :compress, true)
```

Comment:
Could we avoid a keyword lookup here? 
If I remember correctly, this was quite slow and would be executed for every request.

Reply:
I'm not actually going to address this here since I'm planning on doing a broader pass over configuration to be consistent with this everywhere.